google.json

```diff
@@ -386,7 +571,7 @@
     {
         "id": "com.google.android.apps.messaging",
         "label": "Messages by Google",
-        "description": "RCS client from Google, also supports SMS/MMS. Runs in the background.",
+        "description": "RCS client from Google, also supports SMS/MMS. Runs in the background.\nSimple SMS Messenger is a good FOSS replacement based on the AOSP SMS.",
```

Comment:
We don't suggest stuffs in the description. There's a dedicated section for it (which is already filled).